gtags-mode.el

```diff
@@ -278,11 +278,10 @@ name, code, file, line."
 		       (funcall creator
 				(match-string-no-properties 2 line)   ;; name
 				(match-string-no-properties 4 line)   ;; code
-				(concat root (substring-no-properties
-					      line (1+ (match-beginning 1)) (match-end 1))) ;; file
+				(concat root (match-string-no-properties 1 line)) ;; file
 				(string-to-number (match-string-no-properties 3 line))))) ;; line
 		   (apply #'gtags-mode--exec-sync
-		    (append gtags-mode--output-format-options args `(,symbol)) )))
+			  (append `("-C" ,root)  gtags-mode--output-format-options args `(,symbol)) )))
```

Comment:
Please use `--directory` instead of `-C`. It helps with readability

Reply:
Done